src/seraphis_main/tx_component_types.cpp

```diff
@@ -56,7 +56,7 @@ void append_to_transcript(const SpCoinbaseEnoteV1 &container, SpTranscriptBuilde
 {
     transcript_inout.append("core", container.core);
     transcript_inout.append("addr_tag_enc", container.addr_tag_enc.bytes);
-    transcript_inout.append("view_tag", container.view_tag);
+    transcript_inout.append("double_edged_view_tag", container.view_tag.bytes);
```

Comment:
```suggestion
    transcript_inout.append("view_tag", container.view_tag.bytes);
```
Keep tags short and concise where possible.